WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Features/SpeedUpTransactionDialogViewModel.cs

```diff
@@ -88,7 +88,8 @@ private async Task OnSpeedUpTransactionAsync(BuildTransactionResult boostingTran
 			{
 				await Services.TransactionBroadcaster.SendTransactionAsync(boostingTransaction.Transaction);
 				_wallet.UpdateUsedHdPubKeysLabels(boostingTransaction.HdPubKeysWithNewLabels);
-				UiContext.Navigate().To().SendSuccess(_wallet, boostingTransaction.Transaction, NavigationTarget.CompactDialogScreen);
+				var (title, caption) = ("Success", "Your transaction has been successfully accelerated.");
```

Comment:
```suggestion
				var (title, caption) = ("Success", "Your transaction has been successfully speeded up.");
```

Reply:
previously the wording was explicitly changed from _accelerate tx_ to _speed up tx_, so I'd do this for consistency in wording.
("sped up" is also correct english, but "speeded up" looks better to me)

Reply:
NACK, it doesn't feel right.